procedures/foreman/branch.md.erb

```diff
@@ -58,8 +58,9 @@
   * [ ] [foreman-selinux](https://github.com/theforeman/foreman-selinux)
   * [ ] [smart-proxy](https://github.com/theforeman/smart-proxy)
 * [ ] Branch foreman-packaging
-  * [ ] Create rpm/<%= release %> and update `packages/foreman/foreman-release/foreman.gpg`, `mock/*.cfg`, `package_manifest.yaml`, `rel-eng/{releasers.conf,tito.props}`, `repoclosure/*.conf` and `.github/PULL_REQUEST_TEMPLATE.md`
-  * [ ] Create deb/<%= release %> and update `.github/PULL_REQUEST_TEMPLATE.md`
+  * [ ] Create rpm/<%= release %> and update `packages/foreman/foreman-release/foreman.gpg`, `mock/*.cfg`, `package_manifest.yaml`, `rel-eng/{releasers.conf,tito.props}` and `repoclosure/*.conf`
+  * [ ] Create deb/<%= release %>
+  * [ ] Update `.github/PULL_REQUEST_TEMPLATE.md` (by adding `[ ] <%= release %>` to it) in `master`
```

Comment:
can you also remove it from line 22?

Reply:
Hub sometimes also uses it from the actual branch so perhaps we should update it in `master`, `deb/develop` and `rpm/develop`.

Reply:
not only there, not only there

Reply:
(my comment was to tomer, not ewoud :D)

Reply:
hehe, looks like we really wanted to make sure not to miss that ^_^